uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/services/task.js

```diff
@@ -116,10 +116,10 @@ define([
               taskType.arr.push({
                 key: data.values[optionId].key,
                 value: data.values[optionId].value
-              })
-            }
+              });
```

Comment:
You can avoid unnecessary variable creation and nested values fetching by this
```
_.each(data.values, function(option) {
  taskType.arr.push({
    key: option.key,
    value: option.value
  });

  taskType.obj[option.key] = option.value;
})
```
Same should be done below.